configure.ac

```diff
@@ -289,7 +289,11 @@ RKT_ITERATE_FLAVORS([${RKT_STAGE1_FLAVORS}],[flavor],
                                       RKT_COMMON_COREOS_PROGS
                                       RKT_REQ_PROG([PATCH],[patch],[patch])
                                       RKT_REQ_PROG([TAR],[tar],[tar])
-                                      RKT_REQ_PROG([XZ],[xz],[xz])],
+                                      RKT_REQ_PROG([XZ],[xz],[xz])
+                                      AC_CHECK_HEADER([openssl/bio.h],
```

Comment:
Thanks for your commitment :) !
According to [kernel minimal requirements](https://www.kernel.org/doc/Documentation/Changes), you are totally right, but I suggest add this information (+short description and link to documentation) to `Documentation/dependencies.md` file first.